.github/workflows/matrix.yml

```diff
@@ -7,6 +7,7 @@ jobs:
     strategy:
       matrix:
         os: [fedora, debian-x86_64, arch, debian-i386]
+      fail-fast: false
```

Comment:
Why want to change this? The idea is to save CI minutes.

Reply:
Ohh it was just a hack to workaround all the debian builds failing due to a network issue. I've dropped that one now.